homeassistant/components/demo/media_player.py

```diff
@@ -398,6 +407,18 @@ def set_repeat(self, repeat):
         self._repeat = repeat
         self.schedule_update_ha_state()
 
+    async def async_join(self, group_members):
```

Comment:
These methods also need to be defined in the base media player entity class. See other service methods for examples.

Reply:
@MartinHjelmare Thanks for your suggestion. I have now added the two new methods to the base media player class (with a NotImplemented exception).

I also had to rename `async_join` to `async_join_players`, as it turned out that `async_join` was already in use by some other components and pylint did not like that :)

I have also updated the PR description so this is now ready from my side.